README.md

```diff
@@ -129,6 +129,11 @@ Get the `SpanID` of the `activeSpan` in the `Context`.
 go2sky.SpanID(ctx)
 ```
 
+## Reporter
+g2sky does not guarantee that finished spans are reported. Spans that finish just
+before the program exit might not be reported. If you want to ensure that such
+kinds of spans are reported, you can add some sleep time to the main goroutine.
```

Comment:
We don't recommend sleep. I am afraid. Sleeping doesn't guarantee.
Your tracing could still be running.